processing/src/main/java/io/druid/segment/IndexMergerV9.java

```diff
@@ -726,7 +727,8 @@ private void mergeIndexesAndWriteColumns(
   private LongColumnSerializer setupTimeWriter(final IOPeon ioPeon) throws IOException
   {
     LongColumnSerializer timeWriter = LongColumnSerializer.create(
-        ioPeon, "little_end_time", CompressedObjectStrategy.DEFAULT_COMPRESSION_STRATEGY
+        ioPeon, "little_end_time", CompressedObjectStrategy.DEFAULT_COMPRESSION_STRATEGY,
+        CompressionFactory.DEFAULT_LONG_ENCODING
```

Comment:
I think it's fine that it does its own compression thing, as those configs are called dimensionCompression and metricCompression, so, okay, fine, it's not clear which one should apply to __time (it behaves like a dimension but is not stored the same way as "classic" string dimensions).

But __time is definitely a long column, that's for sure, so `longEncoding` should apply to it.